### PR TITLE
Start 2D plotter which builds to desktop and web

### PR DESCRIPTION
**Pull request by frankplus**
Hi there ^.^
Using the already implemented parser I started making a simple 2D plotter using three-d renderer. This builds to both desktop (Rust + OpenGL) and web (Rust to WebAssembly).
- On the web build, the input is taken from an html input tag.
- Implemented zooming with mouse wheel

TODO: 
- translate with mouse drag
- handle vertical asymptotes cases
- draw x/y values on the axis

Future TODO:
- Implicit functions
- 3D functions

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -198,12 +198,12 @@ impl<'s> Parser<'s> {
...
             Token::XVar => {
                 self.next_token();
-                Ok(vec![Operation::Variable(|input| input.0)])
-            },
-            Token::YVar => {
-                self.next_token();
-                Ok(vec![Operation::Variable(|input| input.1)])
+                Ok(vec![Operation::Variable(|input| input)])
             },
+            // Token::YVar => {
```
Comment (Franciman) [inline src/parser.rs:203]: How about we completely remove Token::YVar also from the Tokenizer?

**Review by Franciman — CHANGES_REQUESTED** [review]

**Review by Franciman — APPROVED** [review]
Everything alright, LGTM

File: src/parser.rs (hunk 1 above)
Comment (frankplus) [reply src/parser.rs:203]: Soon we are going to implement plotting 3D  functions so I think we should handle both 2D and 3D functions parsing. let's just keep it there as a placeholder

Comment (Franciman) [reply src/parser.rs:203]: Ok, fine with me.

**Pull request merged**